modules/core/src/main/scala/com.snowplowanalytics.iglu/client/validator/CirceValidator.scala

```diff
@@ -240,4 +248,75 @@ object CirceValidator extends Validator[Json] {
 
   private def fromValidationMessage(m: ValidationMessage): ValidatorReport =
     ValidatorReport(m.getMessage, m.getPath.some, m.getArguments.toList, m.getType.some)
+
+  private def evaluateSchema(
```

Comment:
I like how everything new is now private; you've completely minimised the api interface that we would have to maintain for future versions.  I guess this became possible after you removed the need for a public ValidatorF trait.